src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/aggregation/AggregationSingle.java

```diff
@@ -0,0 +1,51 @@
+package com.alipay.oceanbase.rpc.protocol.payload.impl.execute.aggregation;
+
+import com.alipay.oceanbase.rpc.protocol.payload.impl.execute.aggregation.AggregationType;
+import com.alipay.oceanbase.rpc.util.Serialization;
+
+import static com.alipay.oceanbase.rpc.util.Serialization.encodeObUniVersionHeader;
+import static com.alipay.oceanbase.rpc.util.Serialization.getObUniVersionHeaderLength;
+
+public class AggregationSingle {
+    public AggregationSingle(AggregationType aggregationType_, String column_) {
+        this.agg_column = column_;
+        this.agg_type = aggregationType_;
+    }
+    public int GetTypeNumber() {
```

Comment:
Shall we keep the consistent JAVA code style?
```suggestion
    public int getTypeNumber() {
```
Also, shall we use a better name? Like _TypeIdx_

Reply:
fixed